src/components/StarterPack/QrCodeDialog.tsx

```diff
@@ -77,7 +81,11 @@ export function QrCodeDialog({
       } else {
         setIsProcessing(true)
 
-        if (!AppBskyGraphStarterpack.isRecord(starterPack.record)) {
+        if (
+          !asPredicate(AppBskyGraphStarterpack.validateRecord)(
+            starterPack.record,
+          )
```

Comment:
This might be more readable (& consistent with `dangerousIsType`) if you created another util in the `atp` namespace:

```suggestion
          !atp.safelyValidate(
            starterPack.record,
            AppBskyGraphStarterpack.validateRecord,
          )
```

```ts
// atp.ts

function safelyValidate<T, V = unknown>(
  value: V,
  validate: (value: unknown) => ValidationResult<T>,
): value is T & V {
  return validate(value).success
}
```


WDYT ?

Reply:
Yeah I dig that. I want to go through and rename `#/types/atproto` to bsky in some places, as you suggested as well. Gonna do a pass on that in a sec, good idea to colocate this there too.